config/tools/mqttsource/containersource.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 The Knative Authors
```

Comment:
perhaps some readme, with instructions for MQTT broker on kube etc would be nice?

How about we eventually publish this as a blog post ?

Reply:
Ok I will do that, @matzew can you guide me how test the websockets because similarly I can get the mqtt so that would be batter for me.